stripe/res/values/strings.xml

```diff
@@ -4,48 +4,89 @@
+    <!--Label for input requesting country, used for all locations-->
     <string name="address_label_country">Country</string>
-    <string name="address_label_country_optional">Country (optional)</string>
+    <!--Label for input requesting phone number, used for all locations-->
     <string name="address_label_phone_number">Phone number</string>
+    <!--Label for input requesting phone number where input is optional, used for all locations-->
     <string name="address_label_phone_number_optional">Phone number (optional)</string>
+    <!--Label for input requesting state, used for US based addreses-->
     <string name="address_label_state">State</string>
+    <!--Label for input requesting state where input is optional, used for US based addresses-->
     <string name="address_label_state_optional">State (optional)</string>
+    <!--Label for input requesting zip code, used for US based addreses-->
     <string name="address_label_zip_code">ZIP code</string>
+    <!--Label for input requesting zip code where input is optional, used for US based addresses-->
     <string name="address_label_zip_code_optional">ZIP code (optional)</string>
+    <!--Label for input requesting postal code, used for Canada based addresses-->
     <string name="address_label_postal_code">Postal code</string>
+    <!--Label for input requesting postal code where input is optional, used for Canada based addresses-->
     <string name="address_label_postal_code_optional">Postal code (optional)</string>
+    <!--Label for input requesting postcode, used for UK based addresses-->
     <string name="address_label_postcode">Postcode</string>
+    <!--Label for input requesting postcode where input is optional, used for UK based addresses-->
     <string name="address_label_postcode_optional">Postcode (optional)</string>
+    <!--Label for input requesting postal code, used for international addresses-->
     <string name="address_label_zip_postal_code">ZIP/Postal code</string>
+    <!--Label for input requesting postal code where input is optional, used for international addresses-->
     <string name="address_label_zip_postal_code_optional">ZIP / Postal code (optional)</string>
+    <!--Error text indicating zip code is invalid, used for US addresses-->
     <string name="address_zip_invalid">Your ZIP code is invalid.</string>
+    <!--Error text indicating postcode is invalid, used for UK addresses-->
     <string name="address_postcode_invalid">Your postcode is invalid</string>
+    <!--Error text indicating postal code is invalid, used for Canada addresses-->
     <string name="address_postal_code_invalid">Your postal code is invalid</string>
+    <!--Error text indicating zip/ postal code is invalid, used for international addresses-->
     <string name="address_zip_postal_invalid">Your ZIP/Postal code is invalid</string>
+    <!--Label for input requesting province, used for canadian addresses-->
```

Comment:
Let's pick "Canadian" or "Canada" for consistency. I'd generally be for "Canadian."